spec/dmv_spec.rb

```diff
@@ -23,6 +23,74 @@
     end
   end
 
+  describe '#add_multiple_facilities' do
+    it 'can add CO facilities' do
+      co_dmv_facilities  = DmvDataService.new.co_dmv_office_locations
+      expect(@dmv.facilities).to eq([])
+
+      @dmv.add_multiple_facilities(co_dmv_facilities)
+      expect(@dmv.instance_variable_get(:@facilities).length).to eq(5)
+      expect(@dmv.instance_variable_get(:@facilities)).to all be_a Facility
+
+      expect(@dmv.facilities_offering_service('Vehicle Registration').length).to be 5
+      expect(@dmv.facilities_offering_service('Renew License').length).to be 0
+
+      expect(@dmv.facilities[0].name).to eq('DMV Tremont Branch')
+      expect(@dmv.facilities[0].address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@dmv.facilities[0].phone).to eq('(720) 865-4600')
+      expect(@dmv.facilities[4].services).to eq(['Vehicle Registration'])
+      expect(@dmv.facilities[0].registered_vehicles).to eq([])
+      expect(@dmv.facilities[0].collected_fees).to eq(0)
+      expect(@dmv.facilities[3].hours).to eq('Mon, Tue, Thur, Fri  8:00 a.m.- 4:30 p.m. / Wed 8:30 a.m.-4:30 p.m.')
+    end
```

Comment:
This test and the tests that are similar to it are all okay. One thing you could do in addition to it is check that each element in the array that's returned is a Facility object. You could do something like:

```ruby
facilities = @dmv.add_multiple_facilities(co_dmv_facilities)
facilities.each do |facility|
   expect(facility).to be_an_instance_of(Facility)
end